src/Presentation/DonorDataFormatter.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Presentation;
+
+use WMDE\Fundraising\DonationContext\Domain\Model\Donation;
+use WMDE\Fundraising\PaymentContext\Domain\Model\BankTransferPayment;
+use WMDE\Fundraising\PaymentContext\Domain\Model\DirectDebitPayment;
+use WMDE\Fundraising\PaymentContext\Domain\Model\PaymentMethod;
+
+/**
+ * @license GNU GPL v2+
+ */
+class DonorDataFormatter {
```

Comment:
Currently, I can't think of a better term than `Formatter` (`PresentationExtractor` or `Converter` are not correct either, and `toArrayConverter` exposes too much of the low-level programming language). But formatters in my mind are more for direct output and not for "converting data for presentation". Maybe you can add a comment to the class explains what it does.